Source/WebCore/html/HTMLImageElement.cpp

```diff
@@ -397,13 +397,6 @@ void HTMLImageElement::attributeChanged(const QualifiedName& name, const AtomStr
         if (isInTreeScope() && !m_parsedUsemap.isNull())
             treeScope().addImageElementByUsemap(m_parsedUsemap, *this);
         break;
-    case AttributeNames::compositeAttr: {
-        // FIXME: images don't support blend modes in their compositing attribute.
-        BlendMode blendOp = BlendMode::Normal;
-        if (!parseCompositeAndBlendOperator(newValue, m_compositeOperator, blendOp))
-            m_compositeOperator = CompositeOperator::SourceOver;
```

Comment:
I think this means we can remove `m_compositeOperator` from this class?

Reply:
Let's try.